CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Note: Breaking changes between versions are indicated by "💥".
 
 ## Unreleased
 
+- [Improvement] Add variable `PREVIEW_LMS_BASE` for custom preview domain.
```

Comment:
Please replace "variable" by "configuration setting".

Reply:
Also, this is actually a new feature, so you can replace "Improvement" by "Feature". Please also amend the git commit title accordingly.